src/StreamJsonRpc/MessageFormatterHelper.cs

```diff
@@ -0,0 +1,126 @@
+﻿
+namespace StreamJsonRpc
+{
```

Comment:
Maybe move this into the Reflection sub-namespace to remove it from the common types found in the root namespace.